src/components/Dashboard-Block.js

```diff
@@ -23,28 +26,56 @@ class DashboardBlock extends Component {
 
 	render() {
 		const { title, children, id } = this.props;
-		const { isExpanded } = this.state
+		const { isExpanded } = this.state;
+		const className = `postbox ${ isExpanded ? 'expanded' : 'closed' }`;
+		const onClick = () => this.onToggleExpanded();
 
 		return (
-			<div className={ 'postbox ' + ( isExpanded ? 'expanded' : 'closed' ) } id={ id } >
-				<button type="button" className="handlediv" aria-expanded="true" onClick={ () => this.setState( { isExpanded: ! isExpanded } ) }>
+			<div className={ className } id={ id }>
+				<button type="button" className="handlediv" aria-expanded={ isExpanded } onClick={ onClick }>
 					<span className="screen-reader-text">Toggle panel: { title }</span>
 					<span className="toggle-indicator" aria-hidden="true" />
 				</button>
 				<h2 className="hndle"><span>{title}</span></h2>
-				{ isExpanded && (
-					<div className="inside">
-						{children}
-					</div>
-				) }
+				<div className="inside">
+					{ children }
+				</div>
 			</div>
 		);
 	}
 }
 
 DashboardBlock.propTypes = {
-	id:    PropTypes.string,
-	title: PropTypes.string,
-}
+	id: PropTypes.string.isRequired,
+	title: PropTypes.string.isRequired,
+};
+
+export function withData( { url, ...props } ) {
+	return WrappedComponent => {
+		class WithData extends Component {
+			render() {
+				const { error, loading, ...rest } = this.props;
 
-export default DashboardBlock;
+				if ( loading ) {
+					return <p>Loading...</p>;
+				}
+
+				if ( error ) {
```

Comment:
Somehow, `error` is always `true` even when the request returns 404, and the error object is received as `data`.

Reply:
That's not ideal! It definitely should be passed as `data`, maybe an issue with the cache lookup

Reply:
Resolved via https://github.com/humanmade/platform-ui/pull/37/commits/c32279773fec30149fd4433ba162bfa24ec96422